app/src/main/java/com/gh4a/Gh4Application.java

```diff
@@ -158,6 +166,31 @@ public void onCreate() {
         mServices.put(STAR_SERVICE, new StarService(mClient));
         mServices.put(USER_SERVICE, new UserService(mClient));
         mServices.put(WATCHER_SERVICE, new WatcherService(mClient));
+
+        createNotificationChannels();
```

Comment:
I think it would be better to do this asynchronously to not delay startup for this. I assume you do this instantly to provide the respective controls in the settings app before the first notification arrives? Maybe it also would make sense to do it when enabling notifications in our settings?

Reply:
> Maybe it also would make sense to do it when enabling notifications in our settings?

That's a good idea.